<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Process_BarriereClass</class>
 <widget class="QMainWindow" name="Process_BarriereClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>694</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1140</width>
    <height>694</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1140</width>
    <height>694</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Process_Barriere</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_TitleProcess">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gestion Process</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>461</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="widget_BDDAcess" native="true">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>600</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_BDDAcces">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Accès BDD : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_BDDVerif">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>120</y>
      <width>1151</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="interface_accueil">
     <widget class="QWidget" name="widget_Accueil" native="true">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>20</y>
        <width>1091</width>
        <height>511</height>
       </rect>
      </property>
      <widget class="QWidget" name="widget_FormConnect" native="true">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>621</width>
         <height>401</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_TitleConnexion">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>131</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Connexion</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Login">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>130</y>
          <width>71</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="edit_Login">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>150</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_Mdp">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>190</y>
          <width>101</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mot de passe</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="edit_Mdp">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>210</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_AccesConnexion">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>300</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Accéder</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_ErrorConnect">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>250</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="btn_AccesGestionGlobale">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>210</y>
         <width>171</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>Mode Gestion Globale</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_BienvenueTexte">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>631</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Bienvenue sur l'espace de gestion des modes de la barrière au sein de La Providence.</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="interface_superviseur">
     <widget class="QWidget" name="widget_InfosVehicule" native="true">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>411</width>
        <height>471</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_Immatriculation">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Immatriculation :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_StatutVehicule">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Statut du véhicule : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_TitleInfos">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Informations Véhicule</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_ImmatriculationDisplay">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>160</y>
         <width>251</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_StatutVehiculeDisplay">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>351</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_ChoixModes" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>291</width>
        <height>471</height>
       </rect>
      </property>
      <widget class="QPushButton" name="btn_GestionGlobale">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>240</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Gestion Globale</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_CasparCas">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>130</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cas-par-Cas</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_Manuel">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>360</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Manuel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_TilteChangeMode">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <italic>true</italic>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Changer de mode</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_SupervisionBarriere" native="true">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>230</y>
        <width>351</width>
        <height>251</height>
       </rect>
      </property>
      <widget class="QPushButton" name="btn_OuvertureBarriere">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ouvrir la barrière</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_TitleInfos_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Supervision</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_InfosGlobal" native="true">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>20</y>
        <width>351</width>
        <height>191</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_ActualMode">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Mode actuel :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_StatutBarriere">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Statut de la barrière : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_StatutBarriereDisplay">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>90</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_ActualModeDisplay">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>150</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_VehiculePresence">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Véhicule devant la barrière : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_VehiculePresenceDisplay">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="interface_invite"/>
   </widget>
   <widget class="QWidget" name="widget_SCStatut" native="true">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>491</width>
      <height>91</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_StatutServeur">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Statut serveur :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_StatutClient">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Retour client : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_StatutServeurDisplay">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_StatutClientDisplay">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCasparCas">
   <property name="text">
    <string>Cas-par-Cas</string>
   </property>
  </action>
  <action name="actionGestionGlobale">
   <property name="text">
    <string>Gestion Globale</string>
   </property>
  </action>
  <action name="actionManuel">
   <property name="text">
    <string>Manuel</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Process_Barriere.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_AccesConnexion</sender>
   <signal>clicked()</signal>
   <receiver>Process_BarriereClass</receiver>
   <slot>on_btnAccesConnexion_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_AccesGestionGlobale</sender>
   <signal>clicked()</signal>
   <receiver>Process_BarriereClass</receiver>
   <slot>on_btnAccueilGestionGlobale_cliked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>905</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_OuvertureBarriere</sender>
   <signal>clicked()</signal>
   <receiver>Process_BarriereClass</receiver>
   <slot>on_btnOuvrirBarriere_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>925</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_CasparCas</sender>
   <signal>clicked()</signal>
   <receiver>Process_BarriereClass</receiver>
   <slot>on_btnCasparCas_cliked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_GestionGlobale</sender>
   <signal>clicked()</signal>
   <receiver>Process_BarriereClass</receiver>
   <slot>on_btnGestionGlobale_cliked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Manuel</sender>
   <signal>clicked()</signal>
   <receiver>Process_BarriereClass</receiver>
   <slot>on_btnManuel_cliked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>527</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_btnAccesConnexion_clicked()</slot>
  <slot>on_btnAccueilGestionGlobale_cliked()</slot>
  <slot>on_btnEnvoiDemande_cliked()</slot>
  <slot>on_btnOuvrirBarriere_clicked()</slot>
  <slot>on_btnCasparCas_cliked()</slot>
  <slot>on_btnGestionGlobale_cliked()</slot>
  <slot>on_btnManuel_cliked()</slot>
 </slots>
</ui>
